Vic2ToHoI4/Data_Files/configurables/cultureGroupToGraphics.txt

```diff
@@ -2404,8 +2404,266 @@ sino = {
+			"gfx/leaders/JAP/Portrait_Japan_Generic_3.dds"
+			"gfx/leaders/JAP/Portrait_Japan_Otozo_Yamada.dds"
+		}
+		radical = {
+			"gfx/leaders/Korean_Japan/Portrait_Ainu_Ukaji_Takashi.dds"
+			"gfx/leaders/Asia/Portrait_Asia_Generic_2.dds"
+			"gfx/leaders/JAP/Portrait_Japan_Generic_2.dds"
+		}
+	female_portraits = {
+		military = {
+			"gfx/leaders/Asia/Portrait_Asia_Generic_female_1.dds"
+			"gfx/leaders/North_American_Native/Female/Portrait_Nunavut_Leona_Aglukkaq.dds"
+		}
+		monarch = {
+			"gfx/leaders/Asia/Portrait_Asia_Generic_female_1.dds"
+			"gfx/leaders/North_American_Native/Female/Portrait_Nunavut_Leona_Aglukkaq.dds"
+		}
+		ideological_leader = {
+			"gfx/leaders/Asia/Portrait_Asia_Generic_female_1.dds"
+			"gfx/leaders/North_American_Native/Female/Portrait_Nunavut_Leona_Aglukkaq.dds"
+		}
+	}
+	ideology_minister_portraits = {
+		communism = {
+			"gfx/interface/ideas/idea_sanzo_nosaka.dds"
+		}
+		democratic = {
+			"gfx/interface/ideas/idea_kijuro_shidehara.dds"
+		}
+		fascism = {
+			"gfx/interface/ideas/idea_hitoshi_imamura.dds"
+		}
+		absolutist = {
+			"gfx/interface/ideas/idea_kingoro_hashimoto.dds"
+		}
+		neutrality = {
+			"gfx/interface/ideas/idea_hajime_sugiyama.dds"
+			"gfx/interface/ideas/idea_chuichi_nagumo.dds"
+		}
+		radical = {
+			"gfx/interface/ideas/idea_hino_kumazo.dds"
+			"gfx/interface/ideas/idea_fushimi_hiroyasu.dds"
+		}
+	male_operative_portraits = {
+		"gfx/leaders/Asia/portrait_asia_generic_operative_male_1.dds"
+		"gfx/leaders/Asia/portrait_asia_generic_operative_male_2.dds"
+		"gfx/leaders/Asia/portrait_asia_generic_operative_male_3.dds"
+		"gfx/leaders/Asia/portrait_asia_generic_operative_male_4.dds"
+		"gfx/leaders/Asia/portrait_asia_generic_operative_male_5.dds"
+		"gfx/leaders/Asia/portrait_asia_generic_operative_male_6.dds"
+		"gfx/leaders/Asia/portrait_asia_generic_operative_male_7.dds"
+		"gfx/leaders/Asia/portrait_asia_generic_operative_male_8.dds"
+		"gfx/leaders/Asia/portrait_asia_generic_operative_male_9.dds"
+	}
+	female_operative_portraits = {
+		"gfx/leaders/Asia/portrait_asia_generic_operative_female_1.dds"
+		"gfx/leaders/Asia/portrait_asia_generic_operative_female_2.dds"
+		"gfx/leaders/Asia/portrait_asia_generic_operative_female_3.dds"
+		"gfx/leaders/Asia/portrait_asia_generic_operative_female_4.dds"
+		"gfx/leaders/Asia/portrait_asia_generic_operative_female_5.dds"
+		"gfx/leaders/Asia/portrait_asia_generic_operative_female_6.dds"
+		"gfx/leaders/Asia/portrait_asia_generic_operative_female_7.dds"
+		"gfx/leaders/Asia/portrait_asia_generic_operative_female_8.dds"
+		"gfx/leaders/Asia/portrait_asia_generic_operative_female_9.dds"
+	}
+	male_monarch_portraits = {
+		"gfx/achievements/vojtek_commander.dds" # Iomante 
+	}
+	female_monarch_portraits = {
+		"gfx/achievements/vojtek_commander.dds"
+	}
+	graphical_culture = southamerican_gfx
+	graphical_culture_2d = asian_2d
+}
+
+japonic = {
+	cultures_and_groups = { far_east_asian } # Fallback
```

Comment:
You should have japanese here explicitly, because games from base Vic2 will have it in the east_asian culture group. By making is explicit you'll ensure it gets properly selected.